app/utils.py

```diff
@@ -104,14 +135,16 @@ def gen_data(ckan, pids):
             status = 'Invalid frequency. Status could not be determined.'
 
         data = {
-            'dataset_id': pid,
+            'dataset_id': package['id'],
             'dataset_name': package['name'],
+            'dataset_title': package['title'],
             'last_updated': str(last_updated),
             'needs_update': status in statuses[1:],
             'status': status,
-            'age': age.days,
+            'age': int(age.days),
```

Comment:
I don't see much use for a float in this case. It doesn't bother me, though. If you think it's valuable it can remain a float in the model.